### DISCUSSION TITLE
Script modes documentation

### DISCUSSION THREAD

**Discussion by nyarly** [davatorium/rofi#1266] [Q&A] [answered]
Two questions here:

Is there official docs about how script modes work? I found https://www.mankier.com/5/rofi-script but I can't say whether it's accurate (or _still_ accurate)

There's also some discussion about a streaming scripted mode interface. Is that still a thing?

**Comment by DaveDavenport** [comment, marked as answer]
This is the latest manpage for script mode: https://github.com/davatorium/rofi/blob/next/doc/rofi-script.5.markdown

**Comment by DaveDavenport** [comment]
A more advanced scripting interface is provided by the following plugin: https://github.com/OmarCastro/rofi-blocks
